src/getUsers.js

```diff
@@ -3,3 +3,14 @@ for (let i = 0; i < 100; i++) {
     users.push({ id: `${i}`, name: `User ${i}`, othersClicked: 0, show: true });
 }
 export default users;
+
+// function getUsers() {
```

Comment:
What I expected here was to see an exported method that will encapsulate the logic to create the 100 users. Having globals here like the const `users` you exported will create issues. For example: in UsersList.js method onClickEmitter you are modifying othersClicked counter in a global object that happens to be also stored in state in App.js. So to make it work you added the `clicked` counter to force the component re-render. It gets pretty confusing as you can see :)